### ISSUE TITLE
Does it do logging?

### ISSUE THREAD

**Issue by Kamek437** [simonla/obsidian_waka_box#8]
Does this also log your time, or do I need another plugin for that?

**Comment by simonla** [comment]
No. You can use [obsidian-wakatime](https://obsidian.md/plugins?id=obsidian-wakatime).
